my_labs_solution/lab3_sol.py

```diff
@@ -0,0 +1,70 @@
+
+def add(fnum, snum):
+  try:
+    result = float(fnum) + float(snum)
+  except Exception as e:
+    result = 'N/A'
+    print(e)
+    print('something went wrong')
+  finally:
+    print(f'Result:{result}')
+
+def sub(fnum, snum):
+  try:
+    result = float(fnum) - float(snum)
+  except Exception as e:
+    result = 'N/A'
+    print(e)
+    print('something went wrong')
+  finally:
+    print(f'Result:{result}')
+
+def mul(fnum, snum):
+  try:
+    result = float(fnum) * float(snum)
+  except Exception as e:
+    result = 'N/A'
+    print(e)
+    print('something went wrong')
+  finally:
+    print(f'Result:{result}')
+
+def div(fnum, snum):
+  try:
+    result = float(fnum)/float(snum)
+  except ZeroDivisionError as e:
+    result = 'N/A'
+    print(e)
+    print('snum cannot be zero for operation "/"')
+  except Exception as ex:
+    result = 'N/A'
+    print(ex)
+    print('something went wrong')
+  finally:
+    print(f'Result:{result}')

```

Comment:
consider returning the value from each of the operation methods instead of printing it directly to the console. It may be a more flexible approach.

Reply:
Sure